widgets/js/index.tsx

```diff
@@ -40,6 +40,22 @@ type RenderArgs = {
 export function render({ model, el }: RenderArgs) {
   const componentType = model.get("comp");
 
+  // There is an existing issue where in VS Code it always shows the widget background as white.
+  // (See https://github.com/microsoft/vscode-jupyter/issues/7161)
+  // We tried to fix this in CSS by overridding the style, but there is a race condition whereby
+  // depending on which style gets injected first (ours or ipywidgets), it may or may not work.
+
+  // The solution here is to force our own override to be last in the style list if not already.
+  // It's a bit of a hack, but it works, and I couldn't find something better that wouldn't be fragile.
+
+  if (
+    !el.ownerDocument.head.lastChild?.textContent?.includes("widget-css-fix")
+  ) {
+    const forceStyle = el.ownerDocument.createElement("style");
```

Comment:
I don't have a great understanding of ipywidgets, so I'm not sure. Is `ownerDocument` not something we have control over? Can we not embed a `<style>` tag in the HTML directly?

Reply:
It's not a ipywidgets thing, it's a DOM thing. The element isn't attached to the DOM yet, but every element is associated with an owning document. (You have to call createElement on a document). So this gets the owning document to modify the styles on, and creates a 'style' element that will be associated with that document.